metis-common/metis-common-network/src/test/java/eu/europeana/metis/network/ExternalRequestUtilTest.java

```diff
@@ -0,0 +1,238 @@
+package eu.europeana.metis.network;
+
+import static eu.europeana.metis.network.ExternalRequestUtil.UNMODIFIABLE_MAP_WITH_NETWORK_EXCEPTIONS;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.net.SocketException;
+import java.net.SocketTimeoutException;
+import java.net.UnknownHostException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Stream;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.ServiceUnavailableException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.util.ReflectionTestUtils;
+import org.springframework.web.client.HttpServerErrorException;
+
+/**
+ * Unit test for {@link ExternalRequestUtil}
+ *
+ * @author Jorge Ortiz
+ * @since 01-02-2022
+ */
+class ExternalRequestUtilTest {
```

Comment:
As we discussed with Simon, we are going to treat this matter in another ticket, to reduce the time of this tests. At this moment it takes the default time of 1 second per retry adding up to 30 secs, because of default values.